### PR TITLE
Made some minor changes to typography and language

### PR DESCRIPTION
**Pull request by jonmv**
@kkraune please review and merge :]

### CODE REVIEW

**Comment by jonmv** [comment]
Hmm ... the <code> elements are too red?

**Comment by jonmv** [comment]
Hang on, need more linespacing between items.

File: documentation/vespa-quick-start.html [hunk 1]
```diff
@@ -4,92 +4,121 @@
 ---
 
 <p>
-This article is tested using Docker for Mac - adapt as fit for other platforms.
-Also try the <a href="vespa-quick-start-centos.html">CentOS quick-start</a>.
-</p>
+This article is tested using Docker for Mac &mdash; adapt as you see fit for other platforms.
```
Comment (kkraune) [inline documentation/vespa-quick-start.html]: why &amp;mdash; ?    makes the source code harder to read

**Comment by kkraune** [comment]
yes I agree &lt;code&gt; does not look good.

**Comment by kkraune** [comment]
looks good! merge at will. I like you added linebreaks, makes later diffs much easier to read

File: documentation/vespa-quick-start.html (hunk 1 above)
Comment (jonmv) [inline documentation/vespa-quick-start.html]: Could use —, I suppose, if that works for html.

**Comment by jonmv** [comment]
I'll use <samp> instead of <code>; it's a monospace font without the red. 
I'll change &mdash; to — (it works).
I'll make sure the list looks good, w.r.t. vetical spacing between items. 
Then I'll merge :]

**Comment by jobergum** [comment]
if you change  VESPA_SAMPLE_APPS you should update the linked blog search tutorial.

**Comment by jonmv** [comment]
Good spot, Jo Kristian; I will do that. I was already going through that
document some weeks ago, but then the source got moved in the middle of it.
I'll finish that work when I get my hands on my modified copy (work machine
-- home today with sick child).

**Pull request merged**